src/contracts/LANDRegistry.ts

```diff
@@ -18,8 +18,9 @@ export class LANDRegistry extends Contract {
   static DataError = DataError
 
   constructor(address: string) {
-    super(address, abi)
-    fulfillContractMethods(this, abi)
+    const sanitizedABI = Abi.sanitize(abi)
+    super(address, sanitizedABI)
+    fulfillContractMethods(this, sanitizedABI)
```

Comment:
The code looks 🆗 but shouldn't we do this for every contract (on `Contract` maybe)? or is it more of a specific case kind of thing?

Reply:
I think that we should sanitize ABIs when we need it.

I mean, if we don't have this issue with other contracts, we shouldn't sanitize them